headers/addons/tilt.h

```diff
@@ -0,0 +1,100 @@
+#ifndef _Tilt_H
+#define _Tilt_H
+
+#include "gpaddon.h"
+
+#include "GamepadEnums.h"
+
+#ifndef TILT_ENABLED
+#define TILT_ENABLED 1
+#endif
+
+#ifndef PIN_TILT_1
+#define PIN_TILT_1 -1
+#endif
+
+#ifndef PIN_TILT_2
+#define PIN_TILT_2 -1
+#endif
+
+#ifndef PIN_TILT_FUNCTION
+#define PIN_TILT_FUNCTION -1
+#endif
+
+#ifndef PIN_TILT_LEFT_ANALOG_UP
+#define PIN_TILT_LEFT_ANALOG_UP -1
+#endif
+
+#ifndef PIN_TILT_LEFT_ANALOG_DOWN
+#define PIN_TILT_LEFT_ANALOG_DOWN -1
+#endif
+
+#ifndef PIN_TILT_LEFT_ANALOG_LEFT
+#define PIN_TILT_LEFT_ANALOG_LEFT -1
+#endif
+
+#ifndef PIN_TILT_LEFT_ANALOG_RIGHT
+#define PIN_TILT_LEFT_ANALOG_RIGHT -1
+#endif
+
+#ifndef PIN_TILT_RIGHT_ANALOG_UP
+#define PIN_TILT_RIGHT_ANALOG_UP -1
+#endif
+
+#ifndef PIN_TILT_RIGHT_ANALOG_DOWN
+#define PIN_TILT_RIGHT_ANALOG_DOWN -1
+#endif
+
+#ifndef PIN_TILT_RIGHT_ANALOG_LEFT
+#define PIN_TILT_RIGHT_ANALOG_LEFT -1
+#endif
+
+#ifndef PIN_TILT_RIGHT_ANALOG_RIGHT
+#define PIN_TILT_RIGHT_ANALOG_RIGHT -1
+#endif
+
+#ifndef TILT_SOCD_MODE
+#define TILT_SOCD_MODE SOCD_MODE_NEUTRAL
+#endif
+
+// Tilt Module Name
+#define TiltName "Tilt"
+
+class TiltInput : public GPAddon {
+public:
+	virtual bool available();
+	virtual void setup();       // Tilt Setup
+	virtual void process();     // Tilt Process
+	virtual void preprocess();  // Tilt Pre-Process (Cheat)
+	virtual std::string name() { return TiltName; }
+private:
+	void debounce();
+	uint8_t gpadToBinary(DpadMode, GamepadState);
```

Comment:
The `gpadToBinary` implementation was removed, so this definition for this member can be removed as well.